test/bats/helpers.bash

```diff
@@ -70,5 +70,12 @@ get_ca_cert() {
   if [ $(kubectl get secret -n gatekeeper-system gatekeeper-webhook-server-cert -o jsonpath='{.data.ca\.crt}' | wc -w) -eq 0 ]; then
     return 1
   fi
-  kubectl get secret -n gatekeeper-system gatekeeper-webhook-server-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > $destination
+  kubectl get secret -n gatekeeper-system gatekeeper-webhook-server-cert -o jsonpath='{.data.ca\.crt}' | base64 -d >$destination
+}
+
+compare_generation() {
+  kind="$1"
+  constraint="$2"
+
+  [[ $(kubectl get ${kind}.constraints.gatekeeper.sh ${constraint} -o json | jq '.status.byPod[0].observedGeneration') = $(kubectl get ${kind}.constraints.gatekeeper.sh ${constraint} -o json | jq '.metadata.generation') ]]
```

Comment:
nit: technically each subshell invocation should be in double-quotes to avoid issues with spaces messing up the comparison, practically, generation/observedGeneration have no spaces.

Reply:
added